### PR TITLE
wings: handle single value fields for valkyrie native models

### PR DESCRIPTION
**Pull request by no-reply**
make TransformerValueMapper/OrmConverter treat single value arrays as singular
value objects by default. change the generated fields for ad-hoc valkyrie models
to be Set.of(Anything), by default. don't override existing fields when
generating valkyrie models.

related to #4556

Co-authored-by: Trey Pendragon <tpendragon@princeton.edu>

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: spec/wings/attribute_transformer_spec.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   it "transform the attributes" do
     expect(subject).to include title: work.title,
-                               contributor: work.contributor,
-                               description: work.description
+                               contributor: work.contributor.first,
+                               description: work.description.first
```
Comment (elrayle) [inline spec/wings/attribute_transformer_spec.rb:28]: Looking at GenericWork, it defines `contributor` and `description` without the `multiple:`.  I thought that defaulted to true.  In which case, I would expect these to be multi-valued in the subject.  Am I missing something?

**Review by elrayle — COMMENTED** [review]
Looks really good.  Just one small question.

File: spec/wings/attribute_transformer_spec.rb (hunk 1 above)
Comment (no-reply) [reply spec/wings/attribute_transformer_spec.rb:28]: what you're missing is that this module only creates an attribute hash, not a valkyrie object. the `Set.of(Anything)` dry-types spec is what casts back to a set (if appropriate).

**Review by elrayle — APPROVED** [review]
Thanks for working on this.

**Pull request merged**